api/models/Campaign.js

```diff
@@ -12,6 +12,7 @@ const schema = new mongoose.Schema({
 
   // Properties cached from DS API.
   title: String,
+  run_id: Number,
```

Comment:
None of other Models use a `_id` naming convention -- e.g. `signup.user`, `signup.campaign`.  Let's rename this as `campaign_run`.

If there's a good argument to keep the `_id`, to make it  `campaign_run_id`-- we should change all other properties in other models to keep consistent.